docs/_includes/sidebar.html

```diff
@@ -13,63 +13,62 @@
+      {% assign release = site.github.releases | where_exp: "release", "release.prerelease == false" | sort:
+      "created_at" | reverse | pop %}
+      <a href="{{ release[0].html_url }}">v{{ release[0].tag_name | remove: "@slack/bolt@" }}</a>
     </span>
   </div>
 
   <ul class="sidebar-section">
-    <a href="{{ site.url | append: site.baseurl | append: localized_base_url  }}/tutorial/getting-started"><li class="title">{{ site.t[page.lang].start }}</li></a>
+    <a href="{{ site.url | append: site.baseurl | append: localized_base_url  }}/tutorial/getting-started"
+      ><li class="title">{{ site.t[page.lang].start }}</li></a
+    >
   </ul>
 
   <ul class="sidebar-section">
-    <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#basic"><li class="title">{{ site.t[page.lang].basic }}</li></a>
-    {% assign basic_sections = site.basic | sort: "order" | where: "lang", page.lang %}
-    
-    {% for section in basic_sections %}
-      <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#{{ section.slug }}">
-        <li>{{section.title}}</li>
-      </a>
+    <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#basic"
+      ><li class="title">{{ site.t[page.lang].basic }}</li></a
+    >
+    {% assign basic_sections = site.basic | sort: "order" | where: "lang", page.lang %} {% for section in basic_sections
+    %}
+    <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#{{ section.slug }}">
+      <li>{{section.title}}</li>
+    </a>
     {% endfor %}
   </ul>
 
   <ul class="sidebar-section">
     <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#advanced">
       <li class="title">{{ site.t[page.lang].advanced }}</li>
     </a>
-    {% assign advanced_sections = site.advanced | sort: "order" | where: "lang", page.lang %}
-    {% for section in advanced_sections %}
-      <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#{{ section.slug }}">
-        <li>{{section.title}}</li>
-      </a>
+    {% assign advanced_sections = site.advanced | sort: "order" | where: "lang", page.lang %} {% for section in
+    advanced_sections %}
+    <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#{{ section.slug }}">
+      <li>{{section.title}}</li>
+    </a>
     {% endfor %}
   </ul>
 
-
-  
-  <!--Start (temporary) beta sections. At the moment, there's no ja-jp documentation for this feature-->
-
   <ul class="sidebar-section">
     <a href="{{ site.url | append: site.baseurl | append: localized_base_url }}/concepts#steps">
-      <li class="title"><span class="beta">Beta</span> {{ site.t[page.lang].steps }}</li>
+      <li class="title">{{ site.t[page.lang].steps }}</li>
+    </a>
+    {% assign workflow_steps = site.steps | sort: "order" | where: "lang", page.lang %} {% for step in workflow_steps %}
```

Comment:
hehe I would still advocate to hide the Overview section from the sidebar (and keep it in the main content) to keep the sidebar action-oriented, especially since "Workflow steps" header is clickable and easily navigable to an overview section.

Reply:
I'm totally for this!